packages/esm-sms-app/src/providers/modals/config-upload-modal.component.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
Modal components should use the `.modal.tsx` suffix. So this should be `config-upload.modal.tsx`.